waterlog/src/Components/BtnResolve.js

```diff
@@ -1,57 +1,63 @@
-import React, { Component } from 'react'
+import React, {
+  Component
+} from 'react'
 
- 
 const APIUri = 'http://api.iot.retrotest.co.za/api/monitors';
 class BtnResolve extends Component {
-   
-    monitorInfo = ["id","type","max_flow","location","status"];
-	faults = 0;
+
+  monitorInfo = ["id", "type", "max_flow", "location", "status"];
+  faults = 0;
 
   constructor(props) {
     super(props);
     this.state = {
-			data: [{"id":0,"type":"xxxx","max_flow":0.0,"long":0.0,"lat":0.0,"status":"xxx"}],
+      data: [{
+        "id": 0,
+        "type": "xxxx",
+        "max_flow": 0.0,
+        "long": 0.0,
+        "lat": 0.0,
+        "status": "xxx"
+      }],
       faultyMonitors: 0,
-      text:'Resolve Issue'
-		}; 
-	}
-		
+      text: 'Resolve Issue'
+    };
+  }
+
   componentDidMount() {
     fetch(APIUri)
       .then(response => response.json())
       .then(data => {
-				this.setState({ data });
-			})
+        this.setState({
+          data
+        });
+      })
       .catch(err => {
-          console.log(err)
-			});
-  
-    } 
-  resolve(){ 
-        for(var i = 0; i < this.state.data.length; i++)
-        { 
-          if(this.state.data[i].status==="faulty")
-          {
-           // console.log(this.state.data[i].status);
-           this.setState({
-            text: "Can't resolve "+this.state.data[i].type+" "+ this.state.data[i].id +' is faulty'
-          });
-          }
-          else{
-            this.setState({
-              text: 'Good '
-            });
-          } 
-        }
+        console.log(err)
```

Comment:
Remove the console.log

Reply:
Fixed this

Reply:
is throwing an error appropriate

Reply:
Yes but not to the console log @GiftLanga

Reply:
@laurent-mbuyu This has been changed.